DatadogTrace/Tests/Span/SpanEventBuilderTests.swift

```diff
@@ -410,6 +410,7 @@ class SpanEventBuilderTests: XCTestCase {
 
     func testWhenBuildingSpan_itConvertsTagValuesToString() {
         let builder: SpanEventBuilder = .mockAny()
+        builder.attributesEncoder.outputFormatting = [.sortedKeys]
```

Comment:
can be done in `.mockAny` to avoid repetition?

Reply:
done 👍